x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/normalization/rule_converters.ts

```diff
@@ -560,15 +605,22 @@ interface RuleCreateOptions {
 // eslint-disable-next-line complexity
 export const convertCreateAPIToInternalSchema = (
   input: RuleCreateProps,
+  actionsClient: ActionsClient,
   options?: RuleCreateOptions
 ): InternalRuleCreate => {
   const { immutable = false, defaultEnabled = true } = options ?? {};
 
+  const [systemActions, externalActions] = partition(input.actions, (action) =>
+    actionsClient.isSystemAction(action.id)
+  );
+
   const typeSpecificParams = typeSpecificSnakeToCamel(input);
   const newRuleId = input.rule_id ?? uuidv4();
 
-  const alertActions = input.actions?.map((action) => transformRuleToAlertAction(action)) ?? [];
-  const actions = transformToActionFrequency(alertActions, input.throttle);
+  const alertSystemActions = systemActions?.map((action) => transformRuleToAlertAction(action));
+
+  const alertActions = externalActions?.map((action) => transformRuleToAlertAction(action)) ?? [];
```

Comment:
This mapping can also be extracted out to a reusable function, that gets either `systemActions` or `externalActions` as argeument  and returns them as the `alert` version of them

Reply:
This should be fixed now.